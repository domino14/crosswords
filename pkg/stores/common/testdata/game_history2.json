{
    "events": [
        {
            "nickname": "cesar4",
            "rack": "AEIJNOP",
            "cumulative": 24,
            "row": 7,
            "column": 6,
            "position": "8G",
            "played_tiles": "JAP",
            "score": 24,
            "words_formed": [
                "JAP"
            ],
            "millis_remaining": 53676
        },
        {
            "nickname": "Mina",
            "rack": "BDDEFGT",
            "cumulative": 13,
            "row": 6,
            "column": 7,
            "direction": 1,
            "position": "H7",
            "played_tiles": "B.DGED",
            "score": 13,
            "words_formed": [
                "BADGED"
            ],
            "millis_remaining": 55867
        },
        {
            "nickname": "cesar4",
            "rack": "AEEINOX",
            "cumulative": 46,
            "row": 8,
            "column": 6,
            "position": "9G",
            "played_tiles": "O.EA",
            "score": 22,
            "words_formed": [
                "ODEA",
                "JO",
                "PE"
            ],
            "millis_remaining": 48213
        },
        {
            "nickname": "Mina",
            "rack": "FGLRTTT",
            "cumulative": 20,
            "row": 10,
            "column": 6,
            "position": "11G",
            "played_tiles": "F.LT",
            "score": 7,
            "words_formed": [
                "FELT"
            ],
            "millis_remaining": 45024
        },
        {
            "nickname": "cesar4",
            "rack": "BEINOTX",
            "cumulative": 81,
            "row": 11,
            "column": 3,
            "position": "12D",
            "played_tiles": "BOXE.",
            "score": 35,
            "words_formed": [
                "BOXED",
                "FE"
            ],
            "millis_remaining": 43423
        },
        {
            "nickname": "Mina",
            "rack": "EGNNRTT",
            "cumulative": 28,
            "row": 10,
            "column": 4,
            "direction": 1,
            "position": "E11",
            "played_tiles": "G.N",
            "score": 8,
            "words_formed": [
                "GON"
            ],
            "millis_remaining": 39707
        },
        {
            "nickname": "cesar4",
            "rack": "?GIMNTY",
            "cumulative": 151,
            "row": 13,
            "column": 4,
            "position": "14E",
            "played_tiles": "sTYMING",
            "score": 70,
            "is_bingo": true,
            "words_formed": [
                "STYMING",
                "GONS"
            ],
            "millis_remaining": 37649
        },
        {
            "nickname": "cesar4",
            "rack": "ALDQ?IO",
            "type": 3,
            "cumulative": 156,
            "bonus": 5,
            "millis_remaining": 37712
        },
        {
            "nickname": "Mina",
            "rack": "ENRRSTT",
            "cumulative": 39,
            "row": 9,
            "column": 9,
            "position": "10J",
            "played_tiles": "RENT",
            "score": 11,
            "words_formed": [
                "RENT",
                "ART"
            ],
            "millis_remaining": 32665
        },
        {
            "nickname": "cesar4",
            "rack": "?ADILOQ",
            "cumulative": 206,
            "row": 7,
            "column": 11,
            "direction": 1,
            "position": "L8",
            "played_tiles": "QI.DAr",
            "score": 50,
            "words_formed": [
                "QINDAR"
            ],
            "millis_remaining": 31960
        },
        {
            "nickname": "Mina",
            "rack": "EINRRST",
            "cumulative": 41,
            "row": 11,
            "column": 11,
            "position": "12L",
            "played_tiles": ".R",
            "score": 2,
            "words_formed": [
                "AR"
            ],
            "millis_remaining": 16459
        },
        {
            "nickname": "cesar4",
            "rack": "AEHLNOU",
            "cumulative": 230,
            "row": 12,
            "column": 8,
            "position": "13I",
            "played_tiles": "HOA.",
            "score": 24,
            "words_formed": [
                "HOAR",
                "HI",
                "ON",
                "AG"
            ],
            "millis_remaining": 25517
        },
        {
            "nickname": "Mina",
            "rack": "EINRSTU",
            "cumulative": 109,
            "row": 3,
            "column": 13,
            "direction": 1,
            "position": "N4",
            "played_tiles": "UNITERS",
            "score": 68,
            "is_bingo": true,
            "words_formed": [
                "UNITERS",
                "RENTS"
            ],
            "millis_remaining": 11933
        },
        {
            "nickname": "cesar4",
            "rack": "EILNUVZ",
            "cumulative": 257,
            "column": 14,
            "direction": 1,
            "position": "O1",
            "played_tiles": "VEIN",
            "score": 27,
            "words_formed": [
                "VEIN",
                "UN"
            ],
            "millis_remaining": 17149
        },
        {
            "nickname": "Mina",
            "rack": "ACILRSS",
            "cumulative": 131,
            "row": 4,
            "column": 10,
            "position": "5K",
            "played_tiles": "CAI.S",
            "score": 22,
            "words_formed": [
                "CAINS",
                "VEINS"
            ],
            "millis_remaining": 6701
        },
        {
            "nickname": "cesar4",
            "rack": "ELOUUVZ",
            "cumulative": 273,
            "row": 1,
            "column": 10,
            "position": "2K",
            "played_tiles": "OVUL.",
            "score": 16,
            "words_formed": [
                "OVULE"
            ],
            "millis_remaining": 7623
        }
    ],
    "players": [
        {
            "nickname": "Mina",
            "user_id": "qUQkST8CendYA3baHNoPjk"
        },
        {
            "nickname": "cesar4",
            "user_id": "xjCWug7EZtDxDHX5fRZTLo"
        }
    ],
    "id_auth": "org.aerolith",
    "uid": "ycj5de5gArFF3ap76JyiUA",
    "description": "Created with Macondo",
    "last_known_racks": [
        "DILORSU",
        "EEEKRUZ"
    ],
    "second_went_first": true,
    "challenge_rule": 3,
    "play_state": 2,
    "final_scores": [
        131,
        273
    ],
    "lexicon": "CSW21",
    "winner": 1
}