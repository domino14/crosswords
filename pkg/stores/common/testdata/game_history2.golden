{
  "events":  [
    {
      "rack":  "AEIJNOP",
      "cumulative":  24,
      "row":  7,
      "column":  6,
      "position":  "8G",
      "playedTiles":  "JAP",
      "score":  24,
      "wordsFormed":  [
        "JAP"
      ],
      "millisRemaining":  53676
    },
    {
      "rack":  "BDDEFGT",
      "cumulative":  13,
      "row":  6,
      "column":  7,
      "direction":  "VERTICAL",
      "position":  "H7",
      "playedTiles":  "B.DGED",
      "score":  13,
      "wordsFormed":  [
        "BADGED"
      ],
      "millisRemaining":  55867,
      "playerIndex":  1
    },
    {
      "rack":  "AEEINOX",
      "cumulative":  46,
      "row":  8,
      "column":  6,
      "position":  "9G",
      "playedTiles":  "O.EA",
      "score":  22,
      "wordsFormed":  [
        "ODEA",
        "JO",
        "PE"
      ],
      "millisRemaining":  48213
    },
    {
      "rack":  "FGLRTTT",
      "cumulative":  20,
      "row":  10,
      "column":  6,
      "position":  "11G",
      "playedTiles":  "F.LT",
      "score":  7,
      "wordsFormed":  [
        "FELT"
      ],
      "millisRemaining":  45024,
      "playerIndex":  1
    },
    {
      "rack":  "BEINOTX",
      "cumulative":  81,
      "row":  11,
      "column":  3,
      "position":  "12D",
      "playedTiles":  "BOXE.",
      "score":  35,
      "wordsFormed":  [
        "BOXED",
        "FE"
      ],
      "millisRemaining":  43423
    },
    {
      "rack":  "EGNNRTT",
      "cumulative":  28,
      "row":  10,
      "column":  4,
      "direction":  "VERTICAL",
      "position":  "E11",
      "playedTiles":  "G.N",
      "score":  8,
      "wordsFormed":  [
        "GON"
      ],
      "millisRemaining":  39707,
      "playerIndex":  1
    },
    {
      "rack":  "?GIMNTY",
      "cumulative":  151,
      "row":  13,
      "column":  4,
      "position":  "14E",
      "playedTiles":  "sTYMING",
      "score":  70,
      "isBingo":  true,
      "wordsFormed":  [
        "STYMING",
        "GONS"
      ],
      "millisRemaining":  37649
    },
    {
      "rack":  "ALDQ?IO",
      "type":  "CHALLENGE_BONUS",
      "cumulative":  156,
      "bonus":  5,
      "millisRemaining":  37712
    },
    {
      "rack":  "ENRRSTT",
      "cumulative":  39,
      "row":  9,
      "column":  9,
      "position":  "10J",
      "playedTiles":  "RENT",
      "score":  11,
      "wordsFormed":  [
        "RENT",
        "ART"
      ],
      "millisRemaining":  32665,
      "playerIndex":  1
    },
    {
      "rack":  "?ADILOQ",
      "cumulative":  206,
      "row":  7,
      "column":  11,
      "direction":  "VERTICAL",
      "position":  "L8",
      "playedTiles":  "QI.DAr",
      "score":  50,
      "wordsFormed":  [
        "QINDAR"
      ],
      "millisRemaining":  31960
    },
    {
      "rack":  "EINRRST",
      "cumulative":  41,
      "row":  11,
      "column":  11,
      "position":  "12L",
      "playedTiles":  ".R",
      "score":  2,
      "wordsFormed":  [
        "AR"
      ],
      "millisRemaining":  16459,
      "playerIndex":  1
    },
    {
      "rack":  "AEHLNOU",
      "cumulative":  230,
      "row":  12,
      "column":  8,
      "position":  "13I",
      "playedTiles":  "HOA.",
      "score":  24,
      "wordsFormed":  [
        "HOAR",
        "HI",
        "ON",
        "AG"
      ],
      "millisRemaining":  25517
    },
    {
      "rack":  "EINRSTU",
      "cumulative":  109,
      "row":  3,
      "column":  13,
      "direction":  "VERTICAL",
      "position":  "N4",
      "playedTiles":  "UNITERS",
      "score":  68,
      "isBingo":  true,
      "wordsFormed":  [
        "UNITERS",
        "RENTS"
      ],
      "millisRemaining":  11933,
      "playerIndex":  1
    },
    {
      "rack":  "EILNUVZ",
      "cumulative":  257,
      "column":  14,
      "direction":  "VERTICAL",
      "position":  "O1",
      "playedTiles":  "VEIN",
      "score":  27,
      "wordsFormed":  [
        "VEIN",
        "UN"
      ],
      "millisRemaining":  17149
    },
    {
      "rack":  "ACILRSS",
      "cumulative":  131,
      "row":  4,
      "column":  10,
      "position":  "5K",
      "playedTiles":  "CAI.S",
      "score":  22,
      "wordsFormed":  [
        "CAINS",
        "VEINS"
      ],
      "millisRemaining":  6701,
      "playerIndex":  1
    },
    {
      "rack":  "ELOUUVZ",
      "cumulative":  273,
      "row":  1,
      "column":  10,
      "position":  "2K",
      "playedTiles":  "OVUL.",
      "score":  16,
      "wordsFormed":  [
        "OVULE"
      ],
      "millisRemaining":  7623
    }
  ],
  "players":  [
    {
      "nickname":  "cesar4",
      "userId":  "xjCWug7EZtDxDHX5fRZTLo"
    },
    {
      "nickname":  "Mina",
      "userId":  "qUQkST8CendYA3baHNoPjk"
    }
  ],
  "version":  2,
  "lexicon":  "CSW21",
  "idAuth":  "io.woogles",
  "uid":  "ycj5de5gArFF3ap76JyiUA",
  "description":  "Created with Macondo",
  "lastKnownRacks":  [
    "EEEKRUZ",
    "DILORSU"
  ],
  "challengeRule":  "FIVE_POINT",
  "playState":  "GAME_OVER",
  "finalScores":  [
    273,
    131
  ]
}